contracts/tokens/SecurityToken.sol

```diff
@@ -927,22 +926,21 @@ contract SecurityToken is ERC20, ReentrancyGuard, SecurityTokenStorage, IERC1594
      * @param _to address The address which you want to transfer to
      * @param _value uint256 the amount of tokens to be transferred
      * @param _data The `bytes _data` allows arbitrary data to be submitted alongside the transfer.
-     * @return bool It signifies whether the transaction will be executed or not.
      * @return byte Ethereum status code (ESC)
      * @return bytes32 Application specific reason code
      */
-    function canTransferFrom(address _from, address _to, uint256 _value, bytes calldata _data) external view returns (bool success, byte reasonCode, bytes32 appCode) {
-        (success, reasonCode, appCode) = _canTransfer(_from, _to, _value, _data);
-        if (success && _value > allowance(_from, msg.sender)) {
-            return (false, 0x53, bytes32(0));
+    function canTransferFrom(address _from, address _to, uint256 _value, bytes calldata _data) external view returns (byte reasonCode, bytes32 appCode) {
+        (reasonCode, appCode) = _canTransfer(_from, _to, _value, _data);
+        if ((reasonCode & 0x0F) == 0x01 && _value > allowance(_from, msg.sender)) {
```

Comment:
it should be obvious that this indicates success. could make sense to add either a comment or use the `isSuccess` method you mentioned with the PR details:

> From the ESC-1066 reference implementation we have:
 ```
/**
 * @dev Check if a status code represents success (ie: 0x*1)
 *
 * @param status Binary ERC-1066 status code
 * @return successful A boolean representing if the status code
 *         represents success
 */
function isSuccess(byte status) public pure returns (bool successful) {
    return reasonOf(status) == 0x01;
}
```

also note that https://github.com/PolymathNetwork/polymath-core/pull/690 introduces named status codes and might conflict with this PR (yours is merging to dev-3.0.0; #690 is merged to `audit-fixes` which is pending a merge to dev-3.0.0 (#705))

Reply:
Modified to use a helper `isSuccess` function to check the LS nibble.